Kitodo/pom.xml

```diff
@@ -23,6 +23,28 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     </properties>
 
+    <repositories>
+        <!-- Lets first add the default Maven central repo,
```

Comment:
Should be either "Let us" or "Let's". Maybe also replace "maven" by "Maven" in following line.